src/java.base/share/classes/java/lang/reflect/Method.java

```diff
@@ -567,12 +567,40 @@ public Object invoke(Object obj, Object... args)
     }
 
     /**
-     * Returns {@code true} if this method is a bridge
-     * method; returns {@code false} otherwise.
+     * {@return {@code true} if this method is a bridge
+     * method; returns {@code false} otherwise}
+     *
+     * @apiNote
+     * A bridge method is a {@linkplain isSynthetic synthetic} method created by a Java
+     * compiler alongside a method originating from the source
+     * code. One example use of a bridge method is as technique for a
+     * Java compiler to support <i>covariant overrides</i>, where a
+     * subclass overrides a method and gives the new method a more
+     * specific return type than the method in the superclass. A
+     * common case where covariant overrides are used is for a {@link
+     * java.lang.Cloneable Cloneable} class where the {@code clone}
+     * method inherited from {@code java.lang.Object} is overridden
+     * and declared to return the type of the class; for example
+     * {@link java.util.EnumSet#clone() EnumSet.clone()} returns
+     * {@code EnumSet<E>} rather than {@link java.lang.Object#clone()
+     * Object}. If this technique was being used, the resulting class
+     * file for {@code EnumSet} would have two {@code clone} methods,
+     * one returning {@code EnumSet<E>} and the second a bridge method
+     * returning {@code Object}; the body of the {@code clone} bridge
+     * method calls its non-bridge counterpart and returns its
+     * result. (While the Java language specification forbids a class
+     * declaring two methods with the same parameter types but a
+     * different return type, the virtual machine does not.)
```

Comment:
I'm of two minds about this. This is certainly a good example of a bridge method. It doesn't motivate _why_ a bridge method is necessary in this case. It think it's because the language says you should be able to write code like
```
    EnumSet<E> es2 = es.clone();
```
so there needs to be a method defined in the class file whose return type is assignment-compatible  with `EnumSet`. However, a `clone` method that returns `Object` is the only one that can override the `Object::clone` method. Thus, a something is necessary to span this gap -- a bridge method.

On the other hand, this might be too much detail for here. This is a really obscure location. It seems like somewhere else would be better, and where a full explanation with examples can be provided.